### PR TITLE
Formatting and add editorconfig

### PR DESCRIPTION
**Pull request by BernardoSilva**
- [x] convert missing CRLF line endings to LF
- [x] Add `.editorconfig`
- [x] add script to find CRLF characters on files

### CODE REVIEW

File: bin/find-windows-line-breaks [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Exists with code 1 if a `src` file contains Windows line breaks.
+
+for file in $( find src -type f ); do
+    grep -U $'\015' $file
```
Comment (amyboyd) [inline bin/find-windows-line-breaks]: I made some changes to this file this morning that you might want to copy: https://github.com/partnermarketing/pm2-frontend/commit/ad1b5f633b2243791da42816b4d4a5eb7e9fb9d0

**Comment by antonvas** [comment]
Maybe it makes sense to add `bin/find-windows-line-breaks` to travis?

**Comment by BernardoSilva** [comment]
> Maybe it makes sense to add bin/find-windows-line-breaks to travis?

Added @antonvas

File: bin/find-windows-line-breaks (hunk 1 above)
Comment (BernardoSilva) [inline bin/find-windows-line-breaks]: > I made some changes to this file this morning that you might want to copy: partnermarketing/pm2-frontend@ad1b5f6

File updated @amyboyd

**Pull request merged**